### ISSUE TITLE
Update replication factor in-place

### ISSUE THREAD

**Issue by wiegandf** [Mongey/terraform-provider-kafka#166]
It would be nice if this provider would support updating replication factor in-place. Currently, when increasing the factor from let's say two to three, it will try to recreate the topic:

```
  # kafka_topic.this["topic"] must be replaced
-/+ resource "kafka_topic" "this" {
      ~ id                 = "topic" -> (known after apply)
        name               = "topic"
      ~ replication_factor = 2 -> 3 # forces replacement
        # (2 unchanged attributes hidden)
    }
```

**Comment by Mongey** [comment]
Duplicate of #53.
This is addressed in #148, which will be landing soon(🤞)

**Comment by Mongey** [comment]
Released in `v0.3.0`+
